shared/js/background/events.es6.js

```diff
@@ -539,8 +490,7 @@ browser.alarms.onAlarm.addListener(async alarmEvent => {
     } else if (alarmEvent.name === 'clearExpiredHTTPSServiceCache') {
         httpsService.clearExpiredCache()
     } else if (alarmEvent.name === 'rotateSessionKey') {
-        // TODO fix for manifest v3
-        sessionKey = getHash()
+        utils.resetSessionKey()
```

Comment:
I think we should add an `await` to this call.

Reply:
That doesn't give us the storage guarantees you're expecting as I mentioned in the meeting. I can add it, but it might confuse another contributor expecting they can rely on the API so long as it's always awaited and I don't think this is the case especially with the event listeners.

I think we should address the reliability of the session writing elsewhere and for this the chance of a race condition is if a session can't be written every 24 hours we might risk overwriting it... If we can't rely on that; we have much bigger issues.

Reply:
I think we should add the `await`, but also maybe add a comment to the start of the (already `async`) function to explain that. We already use `await` elsewhere in the function, so I figure we should use `await` consistently, but also take care to explain the potential downfalls to folks.